vendor/github.com/vmware/govmomi/ovf/env.go

```diff
@@ -47,39 +43,26 @@ const (
 	ovfEnvFooter         = `</Environment>`
 )
 
-// A Name represents an XML name (Local) annotated
-// with a name space identifier (Space).
-// In tokens returned by Decoder.Token, the Space identifier
-// is given as a canonical URL, not the short prefix used
-// in the document being parsed.
-type Name struct {
```

Comment:
These were added here so we don't have to import vim25 which contains 100s of thousands of limes. It's a bit hackish but reduces this vendored package significantly.